wolfcrypt/src/asn.c

```diff
@@ -183,6 +183,9 @@ extern int wc_InitRsaHw(RsaKey* key);
     #include <wolfssl/openssl/objects.h>
 #endif
 
+#if defined(HAVE_CRL) && !defined(WOLFCRYPT_ONLY)
+    #include <wolfssl/internal.h>
```

Comment:
This is not allowed. You cannot have the wolfCrypt layer require something from the TLS layer. If someday we split wolfSSL and wolfCrypt this will be a problem. If you need WOLFSSL_CRL then that struct will need to be moved to asn.h and you'll have to do a large refactor. Avoid it please.